draft-ietf-quic-transport.md

```diff
@@ -5456,7 +5460,7 @@ ID can be selected to route later packets to the same server.
 IANA \[SHALL add/has added] a registry for "QUIC Transport Parameters" under a
 "QUIC Protocol" heading.
 
-The "QUIC Transport Parameters" registry governs a 16-bit space.  This space is
```

Comment:
You overdid it here.  It's still 16 for transport parameters.

Reply:
I did not know that, reverted.